src/api/entities/CorporateAction/index.ts

```diff
@@ -135,6 +147,11 @@ export class CorporateAction extends Entity<UniqueIdentifiers> {
    */
   public linkDocuments: ProcedureMethod<LinkCaDocsParams, void>;
 
+  /**
+   * Modify the Corporate Action's checkpoint
+   */
+  public modifyCaCheckpoint: ProcedureMethod<ModifyCaCheckpointParams, void>;
```

Comment:
if I change it, I get this error in the modifyCheckpoint from DividendDistribution

`Property 'modifyCheckpoint' in type 'DividendDistribution' is not assignable to the same property in base type 'CorporateAction'.`

Sounds like a type conflict between the extended class and the same method name in CA.

Reply:
Yeah. For now let's do the following:

- Rename this to `modifyCheckpoint`
- Change the `ModifyCaCheckpointParams` interface so that `checkpoint` isn't optional

This should fix the issue without a type conflict. We will create a task for later on to refactor this so that the signatures can be different without a conflict